src/discrete-layer/components/snack-container/snack-container.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import { useIntl } from 'react-intl';
+import { observer } from 'mobx-react-lite';
+import { Snackbar, SnackbarAction } from '@map-colonies/react-core';
+import { ResponseState } from '../../../common/models/response-state.enum';
+import { useStore } from '../../models/rootStore';
+
+interface SnackDetails {
+  message: string;
+}
+
+export const SnackContainer: React.FC = observer(() => {
+  const { discreteLayersStore } = useStore();
+  const [snackOpen, setSnackOpen] = useState(false);
+  const [snackDetails, setSnackDetails] = useState<SnackDetails>({
+    message: '',
+  });
+  const intl = useIntl();
+  useEffect(() => {
+    switch (discreteLayersStore.state) {
+      case ResponseState.ERROR:
+        setSnackOpen(true);
+        setSnackDetails({
+          message: 'snack.message.failed',
+        });
+        break;
+      case ResponseState.DONE:
+        setSnackOpen(true);
+        setSnackDetails({
+          message: 'snack.message.success',
+        });
+        break;
+      default:
+        break;
```

Comment:
log warn for unsupported Request type?

Reply:
mc-logger still not integrated, worth to open issue for that.
but in this case only in case of DONE and ERROR snack should be shown, otherwise nothing
***Request*** --> ***Response***?

```
enum ResponseState {
  PENDING = 'pending',
  DONE = 'done',
  ERROR = 'error',
  IDLE = 'idle',
}
```

Reply:
open an issue on mc logger

Reply:
opened mc-logger integration issue 
#37 https://github.com/MapColonies/discrete-layer-client/issues/37